src/components/Calendar/doubleCalendar/styled/container.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+const StyledContainer = styled.div`
```

Comment:
Now we put all styled in one file, look here: `src/components/FileSelector/styled/index.js`